utils.go

```diff
@@ -38,13 +38,20 @@ func checkValidMD5(md5 string) ([]byte, error) {
 const (
 	// maximum object size per PUT request is 5GiB
 	maxObjectSize = 1024 * 1024 * 1024 * 5
+	// minimum Part size for multipart upload is 5MB
+	minPartSize = 1024 * 1024 * 5
 )
 
 // isMaxObjectSize - verify if max object size
 func isMaxObjectSize(size int64) bool {
 	return size > maxObjectSize
 }
 
+// Check if part size is less than allowed value.
+func isPartSizeLess(size int64) bool {
```

Comment:
isMinPartSize - 'is minimum part size?" does not convey the meaning.

isPartSizeSmall or isPartSizeLess is better.

similarly `// isMaxObjectSize - verify if max object size` "verify if max object size" sentence is not right.